ONSdigital/response-operations-ui
# [TBL] Call collex directly for reports

# Motivation and Context

Removes the call to backstage for getting chasing reports.

# What has changed



Changes the `download_report` method in collection_exercise_controller.py to hit the collection exercise endpoint directly.

# How to test?



Run the unit tests, make sure they pass.
Run the unit tests, make sure they pass.
Tail the logs for backstage, collection-exercise service. Hit the `response_chasing/<ce_id>/<survey_id>` endpoint using `curl` or Postman. Make sure backstage doesn't get the call, but collection-exercise service does.
